### PR TITLE
FAI-849: Tyrus dashboard

### PR DESCRIPTION
**Pull request by RobGeada**
Introduces the Tyrus bokeh visualizations + TrustyAI runner

### CODE REVIEW

**Review by ruivieira — CHANGES_REQUESTED** [review]
LGTM, thanks @RobGeada!

Just a couple of comments:

- Is it possible to add the `bokeh` dependency to `pyproject.toml`? 
- When running with `tyrus = Tyrus(..., notebook=True)`, both the inline and file visualisations are shown.
   This only seems to happen when, in a notebook, first the file is generated and then the inline is generate in subsequent runs.

File: src/trustyai/utils/tyrus.py [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+         Parameters
+        ----------
+        display = True : boolean
+            Whether to automatically display the dashboard (true) or simply return it (false).
+        """
+        self._generate_saliencies()
+        self._generate_counterfactual_datasource()
+        plots = self._get_plots()
+
+        if display:
```
Comment (ruivieira) [inline src/trustyai/utils/tyrus.py:365]:
```suggestion
        if display:
            return show(plots)

        return plots
```

To fix the linter error.

Comment (RobGeada) [reply src/trustyai/utils/tyrus.py:366]: I've gone for returning `None` otherwise

Comment (RobGeada) [reply src/trustyai/utils/tyrus.py:366]: which did not work, oops

**Review by ruivieira — APPROVED** [review]
LGTM

**Pull request merged**